### PR TITLE
feat(src/components/common): implement navigation, text group, and section text component

### LINKED ISSUE #8: feat(SectionText): create section text component

### LINKED ISSUE #10: feat(TextGroup): create text group component

### LINKED ISSUE #11: feat(Navigation): create navigation component

### PR DESCRIPTION
**Pull request by Jumer-Caragay**
- Closes #8
- Closes #10 
- Closes #11 

I fixed the lint errors by temporarily refactoring some of the code that was already in the repo. We can add back in the code when we have to.
I'm not sure about the naming of the files yet, so if you have any comments on them list them down.

### CODE REVIEW

File: src/components/common/Navigation.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Navigation.module.css';
+
+const Navigation = ({ children }) => (
+  <nav className={styles.navigation}>
+    {children}
```
Comment (selynna) [inline src/components/common/Navigation.jsx:7]: Are you not including `<NavigationItem />` in here?

File: src/components/common/Navigation.module.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (selynna) [inline src/components/common/Navigation.module.css]: is this extra line on purpose?

File: src/components/common/Navigation.module.css [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+.navigation {
+  display: flex;
+  justify-content: center;
+
```
Comment (selynna) [inline src/components/common/Navigation.module.css]: extra line

File: src/components/common/NavigationItem.module.css [hunk 4]
```diff
@@ -0,0 +1,9 @@
+
```
Comment (selynna) [inline src/components/common/NavigationItem.module.css]: extra line?

File: src/components/common/TextGroup.jsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const TextGroup = ({ children }) => (
+  <div>
+    {children}
```
Comment (selynna) [inline src/components/common/TextGroup.jsx:6]: are you not including `<TextHeader ... />` and `<SectionText ... />` here?

File: src/components/common/TextHeader.jsx [hunk 6]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const TextHeader = ({
+  children,
+  fontSize,
+  fontWeight,
+  color,
```
Comment (selynna) [inline src/components/common/TextHeader.jsx:8]: missing headerText (or something similarly named)

File: src/components/common/TextSection.jsx [hunk 7]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const TextSection = ({
+  children,
+  fontWeight,
+  fontSize,
+  color,
```
Comment (selynna) [inline src/components/common/TextSection.jsx:8]: missing paragraphText (or something similarly named)

File: src/components/common/TextHeader.jsx (hunk 6 above)
Comment (Jumer-Caragay) [reply src/components/common/TextHeader.jsx:8]: The headerText comes from children.

File: src/components/common/TextHeader.jsx [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+}) => {
+  const sectionHeader = { fontSize, fontWeight, color };
+  return (
+    <h2 style={sectionHeader}>{children}</h2>
+  );
+};
+
+
```
Comment (selynna) [inline src/components/common/TextHeader.jsx]: extra line

**Review by selynna — CHANGES_REQUESTED** [review]
Looks good, but some extra lines in different places. Will approve after that's fixed.

**Review by selynna — APPROVED** [review]

**Pull request merged**